dojopuzzles/problems/views.py

```diff
@@ -20,3 +20,8 @@ def problem_select(request, problem_id):
     problem.select()
     request.session["problem_selected"] = problem.id
     return redirect(problem)
+
+
+def problem_list(request):
```

Comment:
**issue**: As you mentioned, pagination is not working. Django has a simple built-in support for pagination ([docs here](https://docs.djangoproject.com/en/3.1/topics/pagination/#using-paginator-in-a-view-function)). Hopefully this can help you to implement that.

**issue**: We need tests for this view as well. Use [this file](https://github.com/dojopuzzles/dojopuzzles/blob/master/dojopuzzles/problems/tests/test_views.py) as reference to what tests to include. Please let me know if you need some help writing them.

Reply:
Thanks for the hints, i implement this feature and tests for this view, hope it's good.